L/libunicorn/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libunicorn"
```

Comment:
The upstream project is called "Unicorn":
```suggestion
name = "Unicorn"
```
You also need to rename the directory to match exactly this name

Reply:
thanks, I'll fix that!